### ISSUE TITLE
Agree & implement changes to Contributer setup workflow

### ISSUE THREAD

**Issue by Jimadine** [TabCarousel/TabCarousel#71]
As briefly discussed in https://github.com/TabCarousel/TabCarousel/issues/66#issuecomment-2168248759, the set up steps for contributing code are made slightly more complicated than they need to be. Ideally, one should be able to clone the repo, make code changes and load the unpacked extension in Chrome, and everything should just work. However, a number of changes to file path references need to be made before the extension can be successfully loaded, otherwise the extension will throw errors.

Here is a list of the changes I needed to make:

`src/manifest.json`:
```
-    "service_worker": "background.min.js",
+    "service_worker": "javascripts/background.js",
```
`src/options.html`:
```
-  <script src="options.min.js">
+  <script src="javascripts/options.js">
```
`src/javascripts/options.js`:
```
Replaced entire file content with that from the `dist` `options.js` - I think it related to the import ... lines.
```
`src/javascripts/background.js`:
```
- chrome.action.setIcon({ path: 'images/icon_32_exp_1.75_stop_emblem.png' });
+ chrome.action.setIcon({ path: '../images/icon_32_exp_1.75_stop_emblem.png' });

- chrome.action.setIcon({ path: 'images/icon_32.png' });
+ chrome.action.setIcon({ path: '../images/icon_32.png' });
```
My feeling is that the source code should be changed to the `+` lines and committed to version control, and the necessary file path changes made as part of the build process instead. Otherwise, if things remain as they are, and [as implied by the wiki](https://github.com/TabCarousel/TabCarousel/wiki/Contributing#installing), the code contributer would have to go through [the Release Process](https://github.com/TabCarousel/TabCarousel/wiki/Release-Process) and make code changes to the `dist` files, which seems back-to-front. 

Are there any reasons not to change things as proposed? I'm not familiar with `npm`/`gulp`, so would appreciate input from others.

**Comment by benjaminoakes** [comment]
I agree with the proposed changes and would support patches to make `src` usable for development and ideally release too.  It would be nice if `dist` was unnecessary.

@madhur Any opinion on this?  You are, of course, more familiar with the reasoning behind the current code.
